xwiki-platform-core/xwiki-platform-ldap/xwiki-platform-ldap-authenticator/src/main/java/com/xpn/xwiki/plugin/ldap/XWikiLDAPSearchAttribute.java

```diff
@@ -35,9 +43,15 @@
     /**
      * Attribute value.
      */
+    @Deprecated
```

Comment:
What I was thinking is a new method instead of field. Something like getString.
Converting bytes to String is easy, but reverse conversion is tricky